src/plugins/raven/sagas/connectionFlow.js

```diff
@@ -40,7 +41,7 @@ function* connect(raven) {
  */
 export default function* (raven) {
   const connectsChannel = yield actionChannel(actionTypes.CONNECTION_OPENED, buffers.dropping(1));
-  let attempts = 0;
+  let attempts = 10;
```

Comment:
@tbone1020 leave this at 0 where it was, otherwise it will only attempt a single time